tests/fuzzing/corpus/seed

```diff
@@ -0,0 +1,210 @@
+"""Escape Jupyter magics when converting to other formats"""
```

Comment:
This file reminds me of `magics.py`, do we need to duplicate it, and why that file specifically?

Reply:
@mwouts I can just copy it at build time, I chose it because it had a nice variety of syntax without being too large of a seed. This file is used as the baseline for mutations by the fuzzer